### PR TITLE
+cudf.0.8

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

File: packages/cudf/cudf.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ "DESTDIR=%{prefix}%"
+ "OCAMLLIBDIR=/lib"
+ "BINDIR=/bin"
+ "LIBDIR=/lib"
+ "INCDIR=/include"]
+]
+depends: [
+  "conf-perl" {build}
+  ("extlib" | "extlib-compat")
+  "ocamlbuild" {build}
```
Comment (avsm) [inline packages/cudf/cudf.0.8/opam]: looks like a missing `camlp4` build dependency here.

```
# + ocamlfind ocamldep -package extlib -pp camlp4o -modules cudf_types.mli > cudf_types.mli.depends
# >> Fatal error: OCaml and preprocessor have incompatible versions
# Fatal error: exception Misc.Fatal_error
```

**Comment by vbmithr** [comment]
On 20/03/2017 12:59, Anil Madhavapeddy wrote:
> looks like a missing |camlp4| build dependency here.
> 
> |# + ocamlfind ocamldep -package extlib -pp camlp4o -modules cudf_types.mli >
> cudf_types.mli.depends # >> Fatal error: OCaml and preprocessor have
> incompatible versions # Fatal error: exception Misc.Fatal_error |

The point was actually to avoid the need of camlp4. This package works on my
computer without camlp4, so I thought it was fine, but apparently it does not
pass CI.

Roberto, Stefano: Could you release a new version on cudf trunk: I see that you
deleted more camlp4 stuff after the 0.8 release, and it seems the leftover
things in release 0.8 does not pass CI (though I could build 0.8 without camlp4
on my machine without problem).

Best,
Vincent

**Comment by avsm** [comment]
In this case, it's picking up camlp4 from the base system installation, hence the incompatibility error.

**Comment by vbmithr** [comment]
On 20/03/2017 18:04, Anil Madhavapeddy wrote:
> In this case, it's picking up camlp4 from the base system installation, hence
> the incompatibility error.

Right. I asked them to release a new version, with all mentions to camlp4
deleted. It's already the case in their trunk.

**Comment by avsm** [comment]
thanks! I'll close this one until the new release then.

**Comment by vbmithr** [comment]
On Mon, Mar 20, 2017 at 01:36:01PM +0100, Vincent B. wrote:
> Roberto, Stefano: Could you release a new version on cudf trunk: I see
> that you deleted more camlp4 stuff after the 0.8 release, and it seems
> the leftover things in release 0.8 does not pass CI (though I could
> build 0.8 without camlp4 on my machine without problem).

I've just release libcduf 0.9.

See: https://gforge.inria.fr/projects/cudf/

Cheers.

**Pull request closed without merging**